Matched interns

timrebergen opened the pull request:
Dit is de pagina waarmee studenten hun gematchte stages kunnen inzien. De specifieke benodigde endpoints kloppen nog niet, maar dat is WIP.

timrebergen committed:
Merge branch 'dev' into matched-interns

timrebergen committed:
Updated_search

timrebergen committed:
Merge branch 'matched-interns' of https://github.com/Lumory/Lumory into matched-interns

timrebergen committed:
Styling_aanpassingen_matched_interns

Styling_aanpassingen_matched_interns

timrebergen committed:
Final_CSS_changes_before_push_to_dev

Laatste aantal aanpassingen qua styling.

timrebergen committed:
Merge branch 'dev' into matched-interns

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+  <n-card :bordered="false" class="matched-container" hoverable
+  >
+    <ellipsis-vertical style="float: right" size="20" class="field-header__options-icon"/>
+    <h1 style="font-size: 32px; font-weight: bolder; padding: 0;  margin: 0;">
```
inline styling verplaatsen naar css block, overal

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+  name: "MatchedCard",
+  data() {
+    return {
+      items: [
```
Waarom zitten items in een apart object, en niet bij de internship die je uit de api haalt?

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+    }
+  },
+  methods: {
+    cardClick(index) {
```
Wordt niet gebruikt, kan weg

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+    <br>
+    <br>
+    <div class="tutoring-container" style="display:flex; flex-direction: row">
+      <person-sharp size="20" class="field-header__options-icon"/>
```
class naam klopt niet, is geen options-icon

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+import Container from "./Container";
+import {PersonSharp, EllipsisVertical} from "@vicons/ionicons5";
+import {NTag, NCard, NIcon} from "naive-ui";
+import axios from "axios"
```
Axios en Container worden niet gebruikt

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+        {message: 'Leuke stage'},
+        {message: 'Veel leermogelijkheden'}
+      ],
+      currentHighlightedInternship: ""
```
Wordt niet gebruikt

mitchellvdhut commented on frontend/components/MatchedInterns.vue:
```diff
@@ -0,0 +1,27 @@
+<template>
```
Bestand wordt niet gebruikt

mitchellvdhut commented on frontend/components/ResultCard.vue:
```diff
@@ -0,0 +1,56 @@
+    NScrollbar,
+    NSpace
+  },
+  name: "ResultCard",
```
Naam is vaag, het is geen result maar iets uitgelicht

mitchellvdhut commented on frontend/components/ResultCard.vue:
```diff
@@ -0,0 +1,56 @@
+
+<script>
+import {PersonSharp} from "@vicons/ionicons5";
+import {NCard, NTag, NScrollbar, NSpace, NIcon} from "naive-ui";
```
NScrollbar wordt niet gebruikt

mitchellvdhut commented on frontend/components/MatchedCard.vue:
```diff
@@ -0,0 +1,74 @@
+<template #description>
+  <n-card :bordered="false" class="matched-container" hoverable
+  >
+    <ellipsis-vertical style="float: right" size="20" class="field-header__options-icon"/>
```
Wrap icons met <n-icon></n-icon> voor sizing

mitchellvdhut commented on frontend/components/ResultCard.vue:
```diff
@@ -0,0 +1,56 @@
+      <br>
+      <br>
+      <div class="tutoring-container" style="display:flex; flex-direction: row">
+        <person-sharp size="20" class="field-header__options-icon"/>
```
Wrap icons met n-icon

mitchellvdhut commented on frontend/components/ResultCard.vue:
```diff
@@ -0,0 +1,56 @@
+  },
+  name: "ResultCard",
+  matchedresults: [],
+  currentHighlightedInternship: '',
```
currentHighlightedInternship wordt niet gebruikt

mitchellvdhut commented on frontend/components/ResultCard.vue:
```diff
@@ -0,0 +1,56 @@
+    internship: {}
+  },
+  methods: {
+    cardClick(index) {
```
wordt niet gebruikt

mitchellvdhut commented on frontend/pages/matched.vue:
```diff
@@ -0,0 +1,75 @@
+    <Container class="input-container">
+      <h1 style="font-size: 32px; font-weight:lighter">Gematchte stages</h1>
+    </Container>
+  <div class="underline" style="border-bottom: 1px solid #EFEFEF"/>
```
naive-ui heeft een divider component

mitchellvdhut commented on frontend/pages/matched.vue:
```diff
@@ -0,0 +1,75 @@
+    </Container>
+  <div class="underline" style="border-bottom: 1px solid #EFEFEF"/>
+    <Container class="card-container">
+      <n-scrollbar style="max-height: 700px; padding-right:15px" trigger="none">
```
waarvoor dient de max-height?

mitchellvdhut commented on frontend/pages/matched.vue:
```diff
@@ -0,0 +1,75 @@
+</template>
+<script>
+import Container from "../components/Container";
+import {NList, NListItem, NThing, NTag, NScrollbar, NButton, NInput, NSpace} from "naive-ui";
```
Veel ongebruikte imports

mitchellvdhut commented on frontend/pages/matched.vue:
```diff
@@ -0,0 +1,75 @@
+    axios.get('http://localhost:3001/Company', {
+      headers: {
+        'accept': '/',
+        'Access-Control-Allow-Origin': '<*>'
```
Like i said. CORS was een backend issue. Headers kunnen weg.

mitchellvdhut commented on frontend/pages/matched.vue:
```diff
@@ -0,0 +1,75 @@
+  methods: {
+    cardClick(index) {
+      this.currentHighlightedInternship = index
+      console.log(this.matchedresults[this.currentHighlightedInternship])
```
remove console.logs

mitchellvdhut commented on frontend/pages/matched.vue:
```diff
@@ -0,0 +1,75 @@
+    </Container>
+</template>
+<script>
+import Container from "../components/Container";
```
Ongebruikte import van Container

timrebergen committed:
Feedback_Mitchell_Matched_interns_doorgevoerd

Ik heb de feedback van Mitchell doorgevoerd in de Matched Interns pagina

timrebergen committed:
Merge branch 'dev' into matched-interns

mitchellvdhut committed:
Delete users-bust.svg

mitchellvdhut committed:
Delete MatchedInterns.vue

timrebergen committed:
Removed currenthighlightedinternship

Removed currenthighlightedinternship

timrebergen committed:
Final_changes

timrebergen committed:
Added_error_message

Heb de error handling even aangepast. Als er iets mis is met het laden, krijgt de eindgebruiker een apart kaartje te zien met een foutmelding

timrebergen committed:
Aanpassing_localhost_link

Aanpassing_localhost_link

timrebergen committed:
Deleted plain HTML and reworked with Naive UI components

timrebergen committed:
Merge branch 'dev' into matched-interns
